packages/client/src/raffles/types/postRaffleParticipationResponse.types.ts

```diff
@@ -0,0 +1,6 @@
+import type { PostParticipationData } from '.';
+
+export type PostRaffleParticipationResponse = {
+  data: PostParticipationData;
+  headers: Record<string, string>;
```

Comment:
This prop does not exist on the return value described in swagger. Can you check, please?

Reply:
This file was not being used, so I removed that.